config.yaml

```diff
@@ -69,7 +69,7 @@ features:
     version: "1.0.0"
     tags: ["client: websat"]
     enabled: true
-    auth: false
+    auth: true
```

Comment:
Note, if you do do this, then you have to specify a auth-token in the requests you will make from websat to these endpoints :)

Reply:
Right, yes so I noticed this in the Swagger UI and forgot to ask how auth will work also interacting with this, and for which features we should have `auth: false`